packages/pharos/src/components/link/pharos-link.scss

```diff
@@ -38,6 +38,7 @@
 
 :host([flex]) #link-element {
   display: inline-flex;
+  min-width: 0;
```

Comment:
Not sure this is having the impact I think it's having. Looking into it more...

Reply:
Alright so I think I got to the bottom of it. Check out https://github.com/ithaka/pharos/pull/106/commits/13e2cbb9ac5ad8ab58b993034c11284fa4028317

Basically needed to move the `inline-flex` rule up to the parent in order to properly apply the `min-width` like the SO post suggests. From what I can tell it appears to be behaving as we would like now. Hoping this style modification doesn't lead to any other negative consequences.

Reply:
womp chromatic seems to indicate it visually broke quite a few things

Reply:
I ended up reverting back to what I had originally. In order to make this work this work for us, as the consumer we will need to be `display: flex` on the `pharos-link` itself. This is the only way I have found to get text wrapping to work the way we want without negatively impacting links elsewhere (see previous chromatic).